packages/suite/src/views/settings/SettingsDevice/SettingsDevice.tsx

```diff
@@ -86,10 +88,8 @@ export const SettingsDevice = () => {
 
     const deviceModelInternal = device.features.internal_model;
 
-    const supportsDeviceAuthentication = [
-        DeviceModelInternal.T2B1,
-        DeviceModelInternal.T3T1,
-    ].includes(deviceModelInternal);
+    const supportsDeviceAuthentication = SUPPORTS_DEVICE_AUTHENTICITY_CHECK[deviceModelInternal];
+    const supportFirmwareRevisionCheck = !supportsDeviceAuthentication; // Older devices with no secure element
```

Comment:
```suggestion
    const supportsFirmwareRevisionCheck = !supportsDeviceAuthentication; // Older devices with no secure element
```

Let's list the two devices to avoid some unexpected behaviour with new models.

Reply:
https://github.com/trezor/trezor-suite/pull/13495/commits/5b654e530778665277c72d2d093b910d4ccb91d8